claim/claim.go

```diff
@@ -35,13 +35,13 @@ const (
 // provide the necessary data to upgrade, uninstall, and downgrade
 // a CNAB package.
 type Claim struct {
-	Name       string                 `json:"name"`
-	Revision   string                 `json:"revision"`
-	Created    time.Time              `json:"created"`
-	Modified   time.Time              `json:"modified"`
-	Bundle     *bundle.Bundle         `json:"bundle"`
-	Result     Result                 `json:"result,omitempty"`
-	Parameters map[string]interface{} `json:"parameters,omitempty"`
+	Installation string                 `json:"installation"`
```

Comment:
We put the spec version in bundle.json so tools can tell what version of the spec it has been serialized to match. 

Seeing a breaking change like this and thinking through how tools can detect and migrate existing claims gracefully while also supporting the new claim format, I think we should also start putting the schema version for the claims spec on the `Claim` struct as well.

That way a tool can deserialize to a map or scan the json document for schema version, figure out if it's the old one with `name` or the new one with `installation`, handle the migration if needed, then read it into the new cnab-go struct.

I'll make a new issue in cnab-go for that.

Reply:
Ok, I created #182